server/src/controllers/Events/resolver.ts

```diff
@@ -718,6 +720,7 @@ ${venueDetails}`;
     return event;
   }
 
+  @Authorized(Permission.EventEdit)
```

Comment:
```suggestion
  @Authorized(Permission.EventDelete)
```
After the MVP, we'll likely want users that can modify but not delete events. Since it's only one extra line of config, can we change this?